kyaml/openapi/openapi_benchmark_test.go

```diff
@@ -9,56 +9,16 @@ import (
 
 	openapi_v2 "github.com/google/gnostic/openapiv2"
 	"google.golang.org/protobuf/proto"
-	"k8s.io/kube-openapi/pkg/validation/spec"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi"
-	"sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1218pb"
+	v1212 "sigs.k8s.io/kustomize/kyaml/openapi/kubernetesapi/v1212"
```

Comment:
I wonder if we should use a newer version of the schema. Maybe v1.22?
It can be done in a followup PR.

Reply:
Thanks! Yeah we were thinking about bumping the schema since we are pretty far behind. I filed https://github.com/kubernetes-sigs/kustomize/issues/4583, let's discuss in a followup.